includes/tracks/events/class-wc-coupon-tracking.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * WooCommerce Coupon Tracking
+ *
+ * @package WooCommerce\Tracks
+ */
+
+ /**
+  * This class adds actions to track usage of a WooCommerce Coupon.
+  */
+class WC_Coupon_Tracking {
+	/**
+	 * Init
+	 */
+	public function init() {
+		add_action( 'woocommerce_coupon_object_updated_props', array( $this, 'track_coupon_created' ), 10, 3 );
+	}
+
+	/**
+	 * Send a Tracks event when a coupon is updated.
+	 *
+	 * @param WC_Coupon $coupon        The coupon that has been updated.
+	 * @param Array     $updated_props The props of the coupon that have been updated.
+	 */
+	public function track_coupon_updated( $coupon, $updated_props ) {
```

Comment:
I'm getting an error on this `track_coupon_created` v. `track_coupon_updated`

Reply:
What error are you getting?

Reply:
`WC_Coupon_Tracking' does not have a method 'track_coupon_created'`

I think its a naming discrepancy

Reply:
Whoops, not sure how that happened. This is fixed now.